apollo-router/src/plugins/telemetry/config_new/logging.rs

```diff
@@ -104,12 +105,36 @@ pub(crate) struct StdOut {
     pub(crate) enabled: bool,
     /// The format to log to stdout.
     pub(crate) format: Format,
+    /// Log rate limiting. The limit is set per type of log message
+    pub(crate) rate_limit: Option<RateLimit>,
 }
+
 impl Default for StdOut {
     fn default() -> Self {
         StdOut {
             enabled: true,
             format: Format::default(),
+            rate_limit: None,
+        }
+    }
+}
+
+#[derive(Deserialize, JsonSchema, Clone, Debug)]
+#[serde(deny_unknown_fields, default)]
+pub(crate) struct RateLimit {
+    /// Number of log lines allowed in interval per message
+    pub(crate) count: u32,
+    /// Interval for rate limiting
+    #[serde(deserialize_with = "humantime_serde::deserialize")]
+    #[schemars(with = "String")]
+    pub(crate) interval: Duration,
```

Comment:
I think this should be `period` rather than `interval` because the events happen within a period.

Reply:
in the traffic shaping's rate limiting we use "interval" too, I'll use that to keep consistency

Reply:
Suggest that we use the right terminology and create a migration for traffic shaping. No point in making things confusing for consistency.

Reply:
let's leave that migration to a follow up PR then. I'm not sure I agree that changing it is that useful, because I have seen both terms used in rate limiters, but that discussion should not block this PR (I'd like to get this landed so we can add way more logs in the next release)